### PR TITLE
Msg 75 single movie page

### PR DESCRIPTION
**Pull request by chaomoua91**
### ✨ Why this feature?
- Explain why this feature/fix is needed
when the movie gets clicked on, this shows the details of the movie

### 🛠 Changes being made
styling of the movie info details page

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -1,5 +1,11 @@
-div {
-    /* background: linear-gradient(to bottom, #020617, #334155); */
```
Comment (keenengvang) [inline src/App/App.css:2]: Can we remove this commented out code?

File: src/components/Filter/Filter.css [hunk 2]
```diff
@@ -1,5 +1,12 @@
 .Filter {
-  background: #020617;
+  /* background: linear-gradient(
```
Comment (keenengvang) [inline src/components/Filter/Filter.css]: Can we remove this commented out CSS?

File: src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -2,9 +2,10 @@ import { useEffect, useState } from "react";
 import { tmdbPopularUrl } from "../../constants";
 import axios from "axios";
 import { Movie } from "../../types";
+import { useNavigate } from "react-router-dom";
 import { Card, CardContent } from "@/components/ui/card";
-import { Link } from "react-router-dom";
 import "./Home.css";
+import Filter from "@/components/Filter/Filter";
 // import { Button } from "@/components/ui/button";
```
Comment (keenengvang) [inline src/pages/Home/Home.tsx]: Can we remove these commented out imports?

File: src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -18,6 +19,11 @@ import "./Home.css";
...
 export default function Home() {
   const [movieList, setMovieList] = useState<Movie[]>([]);
+  const navigate = useNavigate();
+
+  const onMovieClick = (movie: Movie) => {
+    navigate(`/info/${movie.id}`);
+  };
 
   useEffect(() => {
     const fetchPopularMovies = async () => {
```
Comment (keenengvang) [inline src/pages/Home/Home.tsx:19]: Can we remove the console.log so that it doesn't appear on production

File: src/pages/InfoPage/InfoPage.tsx [hunk 5]
```diff
@@ -1,11 +1,119 @@
...
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const [movieDetails, movieCredits, movieTrailer] = await Promise.all([
+          axios.get(tmdbMovieDetailsUrl.replace("{movie_id}", id!)),
+          axios.get(tmdbMovieCreditsUrl.replace("{movie_id}", id!)),
+          axios.get(tmdbMovieTrailerUrl.replace("{movie_id}", id!)),
+        ]);
+
+        console.log("Movie Details:", movieDetails.data);
```
Comment (keenengvang) [inline src/pages/InfoPage/InfoPage.tsx]: Can we remove these console logs so they don't appear on production

File: src/App/App.css (hunk 1 above)
Comment (chaomoua91) [reply src/App/App.css:2]: yes

File: src/components/Filter/Filter.css (hunk 2 above)
Comment (chaomoua91) [reply src/components/Filter/Filter.css]: yes

**Comment by keenengvang** [comment]
Looks good!

**Pull request merged**